reference/1.20.4/src/main/generated/.cache/7c60b49dde0390b2089c3d5e64b0bd5954451d13

```diff
@@ -0,0 +1,2 @@
+// 1.20.4	2024-12-25T19:30:31.7452868	Fabric docs reference/Tags for minecraft:damage_type
```

Comment:
Gitignore these files, otherwise they will change whenever someone runs datagen. 1.21.4 will fix this.